src/main/java/com/google/devtools/build/lib/profiler/BUILD

```diff
@@ -12,6 +12,9 @@ java_library(
     name = "profiler",
     srcs = glob([
         "*.java",
+    ], exclude = [
+        # Not used anywhere, but adds dependency to flogger
+        "GoogleAutoProfilerUtils.java",
```

Comment:
this file was open sourced in Oct 2018 by @janakdr , in the hopes that it'd be used more widely in the bazel codebase. looks like no one else on the team decided to use it :(

this BUILD change here might cause some weird merges when we merge this patch internally but that'll be our problem, not yours.

i'm just leaving this comment here for the sake of having a written record of the backstory.